packages/story-editor/src/app/rightClickMenu/provider.js

```diff
@@ -52,6 +52,7 @@ import useDeleteStyle from '../../components/panels/design/textStyle/stylePreset
 import useDeleteColor from '../../components/colorPicker/useDeleteColor';
 import { noop } from '../../utils/noop';
 import useVideoTrim from '../../components/videoTrim/useVideoTrim';
+import isEmptyStory from '../story/utils/isEmptyStory';
```

Comment:
@samwhale Since you have worked on the right-click menu quite a lot and are also reworking on this provider file refactoring currently (#10212) would be great if you could also look take a look at the changes of this PR.

Reply:
Yep, we can extract all the menus in the same way.

Reply:
Yeah this is another way I thought about refactoring the right click menu provider. I think it's also a good way! If y'all have a preference to do it this way rather than what I'm doing in #10212 then lmk

Reply:
Another option apart from these is to create a static menu for each element type. Then we don't have to dynamically generate the right click menu using the right click provider.

@littlemilkstudio what do you think your favorite approach here would be?